peekingduck/runner.py

```diff
@@ -82,9 +84,15 @@ def __init__(
             sys.exit(1)
         if RequirementChecker.n_update > 0:
             sys.exit(3)
+        if num_iter:
+            self.logger.info(f"Run pipeline for {num_iter} iterations")
+            self.num_iter = num_iter
+        else:
+            self.num_iter = 0
```

Comment:
Consider using `is None` or `is not None` when checking a value that can be `None`. And also I think it's better to handle checking 0 value explicitly `== 0` or `!= 0` for clarity. Relevant [google style guide](https://google.github.io/styleguide/pyguide.html#2144-decision).

While the danger of accidentally handling `None` as 0 is not really present here, I think it may be better to just standardize it so we will not have to spend time considering which case is applicable for the comparisons in the future.

Reply:
Agree on this point, current writing feels a bit C-ish. Have rewritten to be clearer and more explicit.